workshops/geoserver/intro/slides/slides.rst

```diff
@@ -60,50 +60,43 @@ Being a community-driven project, GeoServer is developed, tested, and supported
-   :width: 50%
+* Boundless Suite GeoServer Distribution
+* OSGeo Community GeoServer Distribution
 
 Presenter notes
 ---------------
 
-In this section, we will install GeoServer. For the purposes of this workshop, we will be using the OpenGeo Suite—of which GeoServer is a primary component—in order to facilitate setup and configuration.
+In this section, we will install GeoServer.
 
-The OpenGeo Suite is free and open source, and is available for download from Boundless.
+GeoServer is a popular component, this workshop can use your choice of Boundless Suite or OSGeo Community downloads.
 
-OpenGeo Suite is a complete web-based geospatial software stack. In this package, the applications contained are:
+Boundless Suite is available in a standalone virtual machine.
 
-PostGIS - A spatially enabled object-relational database.
-GeoServer - A software server for loading and sharing geospatial data.
-GeoWebCache - A tile cache server that accelerates the serving of maps (built into GeoServer).
-OpenLayers - A browser-based mapping framework
+OSGeo Community downloads will install directly onto your machine; and will require several optional downloads for this workshop.
 
 --------------------------------------------------
 
-Installation
-============
+Boundless Suite
+===============
 
-.. image:: ../doc/source/install/img/installation_welcome.png
+.. image:: ../doc/source/install/img/ecosystem.png
+   :width: 50%
 
 Presenter notes
 ---------------
 
-In this section you will install the OpenGeo Suite on your system. This will provide everything necessary to get started with GeoServer (and more!).
+The Boundless Suite is free and open source, and is available for download from Boundless Connect.
 
-GeoServer, being a Java application, typically requires a Java Runtime Environment (JRE) as well as an application server in order to function. Both of these are included with OpenGeo Suite, so separate installation is not needed here.
+Boundless Suite is a complete web-based geospatial software stack. In this package, the applications contained are:
 
---------------------------------------------------
-
-Installation
-============
-
-.. image:: ../doc/source/install/img/installation_install.png
-
-Presenter notes
----------------
+PostGIS - A spatially enabled object-relational database.
+GeoServer - A software server for loading and sharing geospatial data.
+GeoWebCache - A tile cache server that accelerates the serving of maps (built into GeoServer).
+OpenLayers - A browser-based mapping framework
 
-The OpenGeo Suite installation packages are available for Windows, OS X, Ubuntu, CentOS, and Red Hat. We're using the Windows installers here, but the premise across the board is pretty much the same, and pretty simple: Run the installer, choose options, done.
+Please install VirtualBox, to use the provided virtual machine. During installation pay careful attention to sharing your Desktop as a shared folder.
```

Comment:
Suggest: "To use the provided virtual machine, please install VirtualBox on your system"